ocaml/quicktest/quicktest_crypt_r.ml

```diff
@@ -0,0 +1,209 @@
+(*
+ * Copyright (c) Cloud Software Group, Inc
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+let valid_salts =
+  [
+    "salt"
+  ; (* Don't need to specify algorithm, will default to something. *)
+    "$5$salt$"
```

Comment:
the comments all seem to be off by one, did you mean to put them before the `;`?